pkg/kds/mux/client.go

```diff
@@ -14,10 +14,15 @@ import (
 
 	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
 	"github.com/kumahq/kuma/pkg/config/multizone"
+	"github.com/kumahq/kuma/pkg/core"
 	"github.com/kumahq/kuma/pkg/core/runtime/component"
 	"github.com/kumahq/kuma/pkg/metrics"
 )
 
+var (
+	muxClientLog = core.Log.WithName("kds-mux-client")
```

Comment:
The server log name is `mux-server`. We should be consistent, one way or the other.

Reply:
I believe I've updated the mux-server to be kds-mux-server too.